bin/development-server.js

```diff
@@ -61,9 +54,12 @@ app.use(express.static("public/js/test/examples"));
 app.use(express.static("public"));
 
 // Routes
-
 app.get("/", function(req, res) {
-  res.sendFile(path.join(__dirname, "../index.html"));
+  const tpl = fs.readFileSync(path.join(__dirname, "../index.html"));
+  const html = _.template(tpl)({
```

Comment:
uses lodash to template. there are other template engines, but this is very simple. Also, used lodash below. Didn't take long to add lodash :)